package.json

```diff
@@ -22,26 +22,29 @@
     "build:webpack": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && yarn workspace @compiled/webpack-app build",
     "build:webpack:extract": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && cd examples/packages/webpack && EXTRACT_TO_CSS=true yarn build",
     "bundlesize": "yarn build && size-limit",
-    "clean": "rm -rf node_modules/.cache && find . -name *.tsbuildinfo -type f -delete",
+    "clean": "yarn clean:node-modules && yarn clean:ts-cache && yarn clean:dist",
```

Comment:
Separated clean into four commands, added back dist, fixes up the ts cache one as it needed quotations.